### PR TITLE
Expose contactedReviewers API in v1.0 and misc. updates

### PR DESCRIPTION
**Pull request by shubhamguptacal**
Schema update PR: 5468156

### CODE REVIEW

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+---
+
+# accessReviewInstance resource type---
+title: "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource resource type"
+description: "Represents a principal's access to an access package assignment policy. "
+author: "isabelleatmsft"
+ms.localizationpriority: medium
+ms.prod: "governance"
+doc_type: resourcePageType
+---
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:17]:
```suggestion
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+Represents a principal's access to an access package assignment policy. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows other properties to be passed in.
+
+Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
+| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
+| type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:32]:
```suggestion
| type | String | Type of resource. Type is always `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+|[applyDecisions](../api/accessreviewinstance-applydecisions.md)|None|Manually apply decision on an accessReviewInstance.|
+|[acceptRecommendations](../api/accessreviewinstance-acceptrecommendations.md)|None| Allows the calling user to accept the decision recommendation for each NotReviewed accessReviewInstanceDecisionItem that they are the reviewer on for a specific accessReviewInstance.|
+|[batchRecordDecisions](../api/accessreviewinstance-batchrecorddecisions.md)|None|Review batches of principals or resources in one call.|
+|[filterByCurrentUser](../api/accessreviewinstance-filterbycurrentuser.md)|[accessReviewInstance](../resources/accessreviewinstance.md) collection|Returns all instance objects on a definition for which the calling user is the reviewer.|
+|[List decisions](../api/accessreviewinstance-list-decisions.md)|[accessReviewInstanceDecisionItem](../resources/accessreviewinstancedecisionitem.md) collection|Get the accessReviewInstanceDecisionItem resources from the decisions navigation property.|
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:86]:
```suggestion

```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+The following is a JSON representation of the resource.
+<!-- {
+  "blockType": "resource",
+  "keyProperty": "id",
+  "@odata.type": "microsoft.graph.accessReviewInstance",
+  "baseType": "microsoft.graph.entity",
+  "openType": false
+}
+-->
+``` json
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:112]:
```suggestion

```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+      "@odata.type": "microsoft.graph.accessReviewReviewerScope"
+    }
+  ],
+  "fallbackReviewers": [
+    {
+      "@odata.type": "microsoft.graph.accessReviewReviewerScope"
+    }
+  ]
+}
+```
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:133]:
```suggestion
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+
+Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
+| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
+| type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md:24]:
```suggestion
| type | String | Type of resource. Type will always be `AzureRole`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+Namespace: microsoft.graph
+
+Represents a principal's access to an access package assignment policy. accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource is an open type that allows other properties to be passed in.
+
+Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:30]:
```suggestion
| displayName | String | Display name of the access package. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+Represents access to Azure resource roles. accessReviewInstanceDecisionItemAzureRoleResource is an open type that allows for other properties to be passed in.
+
+Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
+
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md:22]:
```suggestion
| displayName | String | Display name of the Azure role. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+Represents assignment to service principals. accessReviewInstanceDecisionItemServicePrincipalResource is an open type that allows for other properties to be passed in.
+
+Inherits from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| appId | String | The globally unique identifier of the application to which access has been granted. |
+| displayName | String | Display name of the resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md:22]:
```suggestion
| displayName | String | Display name of the service principal. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md).|
```

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+| id | String | Identifier of the decision item resource. Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
+| type | String | Type of resource. Types include: `Group`, `ServicePrincipal`, `DirectoryRole`, `AzureRole`, `AccessPackageAssignmentPolicy`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```
Comment (isabelleatmsft) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md:24]:
```suggestion
| type | String | Type of resource. Type will always be:  `ServicePrincipal`.  Inherited from [accessReviewInstanceDecisionItemResource](accessreviewinstancedecisionItemresource.md). |
```

**Review by isabelleatmsft — APPROVED** [review]

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md [hunk 11]
```diff
@@ -0,0 +1,51 @@
+---
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemaccesspackageassignmentpolicyresource.md:1]: Please sync the details in this file with the details in the beta file.

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md [hunk 12]
```diff
@@ -0,0 +1,50 @@
+---
+title: "accessReviewInstanceDecisionItemAzureRoleResource resource type"
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemazureroleresource.md:2]: Please sync the details in this file with the details in the beta file.

File: api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md [hunk 13]
```diff
@@ -0,0 +1,47 @@
+---
```
Comment (FaithOmbongi) [inline api-reference/v1.0/resources/accessreviewinstancedecisionitemserviceprincipalresource.md:1]: Please sync the details in this file with the details in the beta file.

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 14]
```diff
@@ -1,5 +1,55 @@
...
+          "Target": "accessReviewInstanceDecisionItemAzureRoleResource"
+        },
+        {
+          "Id": "b305ef66-8aaf-4a07-8b0e-7ccbdda4475f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewInstanceDecisionItemServicePrincipalResource",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewInstanceDecisionItemServicePrincipalResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemServicePrincipalResource?view=graph-rest-1.0) resource type.",
+          "Target": "accessReviewInstanceDecisionItemServicePrincipalResource"
+        },
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:28]:
```suggestion
          "Description": "Added the [accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource?view=graph-rest-1.0) resource type. This resource represents an access package assignment policy to which access has been granted through an [accessReviewInstanceDecisionItem]((https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-1.0) object.",
          "Target": "accessReviewInstanceDecisionItemAccessPackageAssignmentPolicyResource"
        },
        {
          "Id": "b305ef66-8aaf-4a07-8b0e-7ccbdda4475f",
          "ApiChange": "Resource",
          "ChangedApiName": "accessReviewInstanceDecisionItemAzureRoleResource",
          "ChangeType": "Addition",
          "Description": "Added the [accessReviewInstanceDecisionItemAzureRoleResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemAzureRoleResource?view=graph-rest-1.0) resource type. This resource represents Azure resource roles to which access has been granted through an [accessReviewInstanceDecisionItem]((https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-1.0) object.",
          "Target": "accessReviewInstanceDecisionItemAzureRoleResource"
        },
        {
          "Id": "b305ef66-8aaf-4a07-8b0e-7ccbdda4475f",
          "ApiChange": "Resource",
          "ChangedApiName": "accessReviewInstanceDecisionItemServicePrincipalResource",
          "ChangeType": "Addition",
          "Description": "Added the [accessReviewInstanceDecisionItemServicePrincipalResource](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstanceDecisionItemServicePrincipalResource?view=graph-rest-1.0) resource type. This resource represents service principals to which access has been granted through an [accessReviewInstanceDecisionItem]((https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitem?view=graph-rest-1.0) object.",
          "Target": "accessReviewInstanceDecisionItemServicePrincipalResource"
        },
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 15]
```diff
@@ -1,5 +1,55 @@
...
+        {
+          "Id": "b305ef66-8aaf-4a07-8b0e-7ccbdda4475f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "contactedReviewers",
+          "ChangeType": "Addition",
+          "Description": "Added the **contactedReviewers** relationship to [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-1.0) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:34]:
```suggestion
          "Description": "Added the **contactedReviewers** relationship to the [accessReviewInstance](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewInstance?view=graph-rest-1.0) resource.",
```

File: changelog/Microsoft.IdentityGovernance.AccessReviews.json [hunk 16]
```diff
@@ -1,5 +1,55 @@
...
+          "Target": "accessReviewInstance"
+        },
+        {
+          "Id": "b305ef66-8aaf-4a07-8b0e-7ccbdda4475f",
+          "ApiChange": "Resource",
+          "ChangedApiName": "accessReviewReviewer",
+          "ChangeType": "Addition",
+          "Description": "Added the [accessReviewReviewer](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewReviewer?view=graph-rest-1.0) resource",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.IdentityGovernance.AccessReviews.json:42]:
```suggestion
          "Description": "Added the [accessReviewReviewer](https://docs.microsoft.com/en-us/graph/api/resources/accessReviewReviewer?view=graph-rest-1.0) resource.",
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @isabelleatmsft, @shubhamguptacal—I've proposed a few changes, mostly to align these changes to decision items API with the changes you're making through [the PR that changes the beta docs](https://github.com/microsoftgraph/microsoft-graph-docs/pull/15036).

Also, please make the changes to the description of [accessReviewInstanceDecisionItemResource](https://docs.microsoft.com/en-us/graph/api/resources/accessreviewinstancedecisionitemresource?view=graph-rest-1.0). That's the cord that ties these 3 new resource files to existing files.

**Review by FaithOmbongi — APPROVED** [review]
Hi @shubhamguptacal, @isabelleatmsft, pushed a few commits to sync the decisionItemResource inherited types to the changes in PR #15036. This PR is also good to go.

**Pull request merged**